linode/manager
# chore: Clean up unused assets & icons

## Description 📝

- Clean up unused assets & icons 🧹 
- Trying to pad my "number of lines deleted" stats 🗑️➖😅
- This all started by me noticing `packages/manager/src/assets/icons/cloudpulse.svg` was unused. I figured I might as well do a quick clean up. We will likely be pulling all future icons from CDS 2.0 🎨 

## How to test 🧪

> [!warning]
> There is no type-safety for assets like SVGs, **but** the Vite build will fail if an imported SVG does not exist

- Verify all checks pass ✅ 
- Checkout this PR or visit the preview link
- Visually scan Cloud Manager for missing assets/icons 👀 

## As an Author I have considered 🤔
